### ISSUE TITLE
[BUG] [Go service][Linux]Can't get ES-419(upper-case with number) locale's translation when service is running on linux OS.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#1909]
**Describe the bug**
commit: 7aeb07348a137caaef30f7c86ebece81207e0e38
Can't get ES-419(upper-case with number) locale's translation when service is running on Linux OS.
It is reproducible on All translation's APIs.

**To Reproduce**
1. Start service on Linux OS and have messages_es-419.json file on bundles folder.
2. Take v2 component api as example: 
`http://<ip:port>/i18n/api/v2/translation/products/VMwareVIP2018/versions/1.1.0/locales/ES-419/components/default?pseudo=false&checkTranslationStatus=false`
3. See error:
![image](https://user-images.githubusercontent.com/51813046/194789870-ea50645e-20a8-4595-9557-13dbdb08df81.png)

**Expected behavior**
It should get the translation from messages_es-419.json file.
```
{
  "response": {
    "code": 200,
    "message": "OK"
  },
  "data": {
    "productName": "VMwareVIP2018",
    "version": "1.1.0",
    "locale": "es-419",
    "component": "default",
    "messages": {
      "Dear_Custom": "Estimado Custom",
      "Partner_Name": "Nombre del socio"
    }
  }
}
```

**Comment by lyiyu66** [comment]
On Windows OS, the translation can be returned but the locale is ES-419. It should be formatted locale: es-419.
Actual behavior:
```
{
  "response": {
    "code": 200,
    "message": "OK"
  },
  "data": {
    "productName": "VMwareVIP2018",
    "version": "1.1.0",
    "locale": "ES-419",
    "component": "default",
    "messages": {
      "Dear_Custom": "Estimado Custom",
      "Partner_Name": "Nombre del socio",
      "Verify_Translation": "Verificar tradução"
    }
  }
}
```

**Comment by lyiyu66** [comment]
the issue is fixed with commit b39601080f6ad4b9158f1678af741dacaaccccc8
